filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -3176,14 +3175,14 @@ void OpenGLDriver::updateTextureLodRange(GLTexture* texture, int8_t targetLevel)
         if (targetLevel < texture->gl.baseLevel || targetLevel > texture->gl.maxLevel) {
             bindTexture(OpenGLContext::MAX_TEXTURE_UNIT_COUNT - 1, texture);
             gl.activeTexture(OpenGLContext::MAX_TEXTURE_UNIT_COUNT - 1);
-            if (targetLevel < texture->gl.baseLevel) {
+            /*if (targetLevel < texture->gl.baseLevel) {
```

Comment:
a ticket number would be nice in comments for these types of "fixes", (same in `setMinMaxLevels()`)